src/audio/dai.c

```diff
@@ -10,6 +10,7 @@
 #include <sof/audio/pipeline.h>
 #include <sof/common.h>
 #include <sof/debug/panic.h>
+#include <sof/drivers/edma.h>
 #include <sof/drivers/ipc.h>
```

Comment:
separate dai.c changes in another commit.

Reply:
Done.